tests/containers/aboutTheData/AgencyDetailsContainer-test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, waitFor, fireEvent, screen } from 'test-utils';
+import '@testing-library/jest-dom/extend-expect';
+import * as aboutTheDataHelper from 'helpers/aboutTheDataHelper';
+import AgencyDetailsContainer from 'containers/aboutTheData/AgencyDetailsContainer';
+import { mockReportingPeriodRow } from './mockData';
+
+const defaultProps = {
+    agencyName: 'Mock Agency (ABC)',
+    agencyCode: '123',
+    modalClick: jest.fn()
+};
+
+const mockResponse = {
+    promise: new Promise((resolve) => {
+        process.nextTick(() => {
+            resolve(mockReportingPeriodRow);
+        });
+    }),
+    cancel: () => {
+        console.log('cancel executed!');
+    }
+};
+
+test('makes a request for table data on mount', () => {
+    const tableRequest = jest.spyOn(aboutTheDataHelper, 'fetchAgency').mockReturnValue(mockResponse);
+
+    render(<AgencyDetailsContainer {...defaultProps} />);
+    expect(tableRequest).toHaveBeenCalledTimes(2); // once for each useEffect
```

Comment:
isn't this a duplicate request?

Reply:
I think this is the same issue we were talking about on a call last week. It seems to register once for each `useEffect` in the unit tests, but I'm not seeing it duplicated in the browser itself.